jinseok/db/migrations/20230419013034_create_posts_table.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE posts (
+    post_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    text VARCHAR(200) NOT NULL,
+    user_id INT NOT NULL,
+    FOREIGN KEY (user_id) REFERENCES users (id)
+)
```

Comment:
```suggestion
CREATE TABLE posts (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    text VARCHAR(200) NOT NULL,
    user_id INT NOT NULL,
    FOREIGN KEY (user_id) REFERENCES users (id)
)
```
posts table의 primary key 컬럼의 이름은 post_id가 아니라 id가 되어야합니다.

추가로 가급적 최대한 어떠한 참고없이 스스로 구현해보고, 어려움이 있으면 찾아와주세요!